cfgmgr/buffermgrdyn.cpp

```diff
@@ -2456,12 +2478,21 @@ task_process_status BufferMgrDynamic::handleBufferProfileTable(KeyOpFieldsValues
             {
                 profileApp.size = value;
             }
-            else if (field == buffer_dynamic_th_field_name)
-            {
-                profileApp.threshold = value;
-            }
-            else if (field == buffer_static_th_field_name)
+            else if (field == buffer_dynamic_th_field_name || field == buffer_static_th_field_name)
             {
+                if (profileApp.threshold_mode.empty())
+                {
+                    profileApp.threshold_mode = field;
```

Comment:
No. The idea is
- `threshold_mode` can be deducted from either the buffer pool it referenced or the name of the corresponding field
- both should be the same
- we take the one received first and check whether the second one is the same as the first one. in case they differ, an error message will be logged, which is implemented in the `else if (profileApp.threshold_mode != field)` branch